examples/ubuntu/ubuntu-2004.pkr.hcl

```diff
@@ -0,0 +1,86 @@
+packer {
+  required_plugins {
+   xenserver= {
+      version = ">= v0.3.2"
+      source = "github.com/ddelnano/xenserver"
+    }
+  }
+}
+
+variable "remote_host" {
+  type        = string
+  description = "The ip or fqdn of your XenServer. This will be pulled from the env var 'PKR_VAR_XAPI_HOST'"
+  sensitive   = true
+  default     = null
+}
+
+variable "remote_password" {
+  type        = string
+  description = "The password used to interact with your XenServer. This will be pulled from the env var 'PKR_VAR_XAPI_PASSWORD'"
+  sensitive   = true
+  default     = null
+}
+
+variable "remote_username" {
+  type        = string
+  description = "The username used to interact with your XenServer. This will be pulled from the env var 'PKR_VAR_XAPI_USERNAME'"
+  sensitive   = true
+  default     = null
+
+}
+
+variable "sr_iso_name" {
+  type        = string
+  default     = ""
+  description = "The ISO-SR to packer will use"
+
+}
+
+variable "sr_name" {
+  type        = string
+  default     = ""
+  description = "The name of the SR to packer will use"
+}
+
+locals {
+  timestamp = regex_replace(timestamp(), "[- TZ:]", "") 
+}
+
+
+source "xenserver-iso" "ubuntu-2004" {
+  iso_checksum      = "443511f6bf12402c12503733059269a2e10dec602916c0a75263e5d990f6bb93"
+  iso_checksum_type = "sha256"
+  iso_url           = "http://releases.ubuntu.com/20.04/ubuntu-20.04.1-live-server-amd64.iso"
+
+  sr_iso_name    = var.sr_iso_name
+  sr_name        = var.sr_name
+  tools_iso_name = "guest-tools.iso"
+
+  remote_host     = var.remote_host
+  remote_password = var.remote_password
+  remote_username = var.remote_username
+
+  vm_name        = "packer-ubuntu-2004-${local.timestamp}"
+  vm_description = "Build started: ${local.timestamp}"
+  vm_memory      = 4096
+  disk_size      = 20000
+
+  http_directory = "examples/http/ubuntu-2004"
+  boot_command   = [
+    "<esc><f6> autoinstall ds=nocloud-net;s=http://{{ .HTTPIP }}:{{ .HTTPPort }}/<enter><wait>",
+    "<f6><wait><esc><wait> autoinstall ds=nocloud-net;s=http://{{ .HTTPIP }}:{{ .HTTPPort }}/<enter><wait>"
+  ]
+  boot_wait      = "10s"
+
+  ssh_username     = "testuser"
+  ssh_password     = "ubuntu"
+  ssh_wait_timeout = "10m"
+  shutdown_command = "/sbin/shutdown"
```

Comment:
Can we remove the `shutdown_command` from all the examples? Judging from #19, this is likely erroring out and relying on the XAPI vm shutdown api anyway. The docs were updated in that issue to indicate that this config key should be for exceptional cases only and that the default should work for most usecases.

Reply:
I didn't know about #19, will do